docs/src/main/paradox/release-notes.md

```diff
@@ -2,6 +2,13 @@
 
 ## 0.9.x
 
+### 0.9.1
+
+* Added `method_name` parameter to the `rf_resample` method.
+   * __BREAKING__: In SQL, the function `rf_resample` now takes 3 arguments. You can use `rf_resample_nearest` with two arguments or refactor to `rf_resample(t, v, "nearest")`.
+* Added resample method parameter to SQL and Python APIs. This will affect the reprojection of right hand side tiles.
```

Comment:
> This will affect the reprojection of right hand side tiles.
 
Can you be explicit in what way here?

Reply:
maybe better to discuss in the raster join chapter of docs?

Reply:
take a look here